src/components/Filter/Filter.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { useFilter } from '../../context/filter/filterContext';
+import './Filter.css';
+function Filter() {
+  const [hidden, setHidden] = useState(true);
+  const { filterState, filterDispatch } = useFilter();
+  console.log(filterState);
+  return (
+    <div className="filter-container">
+      <button className="btn" onClick={() => setHidden(!hidden)}>
+        Filter
+      </button>
+      <div className={hidden ? 'filter' : 'filter show'}>
+        <div>Sort By</div>
+        <div>
+          <input
+            id="latest-first"
+            type="radio"
+            checked={filterState.sortBy === 'LATEST_FIRST'}
+            onChange={e => filterDispatch({ type: 'LATEST_FIRST' })}
```

Comment:
What is the need of passing `e` as an argument here? I think you could do it without that as well i.e. 
```jsx
onChange={()=> filterDispatch({type: "LATEST_FIRST"})}
```